pkg/controller/operators/operatorcondition_controller_test.go

```diff
@@ -313,6 +313,42 @@ var _ = Describe("OperatorCondition", func() {
 							Value: operatorCondition.GetName(),
 						}))
 					}
+
+					// Remove the container's Environment Variables defined in the deployment
```

Comment:
Non-blocking nit: instead of using comments, you can use [`By()`](https://onsi.github.io/ginkgo/#documenting-complex-its-by) so that the step that fails within the `It` block can be printed in the test output.